src/pseudopeople/interface.py

```diff
@@ -203,15 +234,41 @@ def _get_data_changelog_version(changelog: Path) -> Version:
     return version
 
 
+@overload
 def generate_decennial_census(
     source: Path | str | None = None,
     seed: int = 0,
     config: Path | str | dict[str, Any] | None = None,
     year: int | None = 2020,
     state: str | None = None,
     verbose: bool = False,
-    engine: Literal["pandas", "dask"] = "pandas",
+    engine: Literal["pandas"] = "pandas",
 ) -> pd.DataFrame:
+    ...
+
+
+@overload
+def generate_decennial_census(
+    source: Path | str | None,
+    seed: int,
+    config: Path | str | dict[str, Any] | None,
+    year: int | None,
+    state: str | None,
+    verbose: bool,
+    engine: Literal["dask"],
+) -> dd.DataFrame:
+    ...
```

Comment:
Note: I'm curious how overloads look in the docs, will take a look at this after I finish reviewing the code.

Reply:
The overloads don't show up 😞 Oh well, at least they could be helpful for autocomplete etc

Reply:
Hmm, that's too bad. I didn't think to check myself but sphinx [def worked on it](https://github.com/sphinx-doc/sphinx/commit/fb2f777079506a6ed36077ae98c2b267d25b16b5).